content/01-responsive-web-design/applied-visual-design/create-visual-balance-using-the-text-align-property.md

```diff
@@ -1,6 +1,6 @@
 ---
 id: 587d7791367417b2b2512ab3
-title: Create Visual Balance Using the text-align Property
+title: Criar um equilíbrio visual (Visual Balance) usando a propriedade 'text-align'
```

Comment:
I think it would be nice if you put "Crie" instead of "Criar". When I read the titles on freeCodeCamp, I have a personal impression to read first verb in imperative tense... as commands lines. It's just a suggestion... 😊

Reply:
I have this feeling too, thanks for the suggestion 😄